### PR TITLE
global: travis.yml addition

### LINKED ISSUE #5: global: add travis configuration file
Add a fresh travis configuration file `.travis.yml`.

### PR DESCRIPTION
**Pull request by hachreak**
* Adds a fresh .travis.yml file. (closes #5)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+python:
+  - "2.7"
+
+install:
+  - travis_retry pip install -e .[docs]
+
+script:
+  - sphinx-build -qnNW docs docs/_build/html
+  - python setup.py test -a "tests --cov-config .coveragerc --cov kwalitee --cov-report term-missing"
```
Comment (jirikuncar) [inline .travis.yml]: move `"tests --cov-config .coveragerc --cov kwalitee --cov-report term-missing"` correctly to `pytest.ini` (see invenio-tags/pytest.ini)

**Comment by hachreak** [comment]
@jirikuncar I think that we can merge and create a issue for code coverage and investigate `bibsort` problems on invenio initialization (if it's a true issue for us or not).
What do you think about that?

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[tox]
```
Comment (jirikuncar) [inline tox.ini]: missing license

File: setup.py [hunk 3]
```diff
@@ -104,6 +106,7 @@ def run_tests(self):
             'Sphinx',
             'sphinx_rtd_theme'
         ],
+        'devs': test_requirements
```
Comment (jirikuncar) [inline setup.py]: `dev`, `devel`, `development` or simply `test`?

File: tox.ini (hunk 2 above)
Comment (hachreak) [inline tox.ini]: done! :)

File: pytest.ini [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+[pytest]
+addopts = --clearcache --pep8 --ignore=docs --cov=invenio_knowledge --cov-report=term-missing
+pep8ignore =
+    tests/* ALL)
```
Comment (jirikuncar) [inline pytest.ini]: `tests/* ALL)` or `tests/* ALL`?

**Pull request merged**